hack/ensure-gh-cli.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
If we decide to break this up into individual scripts, I have been trying to clean up the hack directory... if we do this, we should probably probably move these individual scripts into a `ensure-deps` folder.

Reply:
Ah okay. If you are already doing this, then I'll leave it out of this PR. Does that make sense? I think the same could be said about https://github.com/vmware-tanzu/community-edition/pull/2570#discussion_r752434686

Reply:
I'm not currently doing this work. I was saying if we want to break them apart, we should do it for everything. In the mean time until that work happens, can you move this into a subfolder called `ensure-deps`.